README.md

```diff
@@ -20,6 +23,7 @@ Install dependencies:
 * pango
 * cairo
 * systemd or elogind (for the sd-bus library)
+* dbus (with user-session support)
```

Comment:
Isn't this a transitive dependency of systemd/elogind?

Reply:
Not on Gentoo at least. I could install and run sway with elogind without any issues when dbus didn't have user-session support. (Systemd doesn't require user session support on dbus either)

It was only when I tried to run mako that I ran into issues. And it took quite a while to figure out what I needed to do, so having this would have been very useful to me.

Reply:
That's true.